components/inx/server_utxo.go

```diff
@@ -244,34 +246,34 @@ func (s *Server) ListenToLedgerUpdates(req *inx.SlotRangeRequest, srv inx.INX_Li
 	// if a startIndex is given, we send all available milestone diffs including the start index.
 	// if an endIndex is given, we send all available milestone diffs up to and including min(ledgerIndex, endIndex).
 	// if no startIndex is given, but an endIndex, we don't send previous milestone diffs.
-	sendPreviousStateDiffs := func(startIndex iotago.SlotIndex, endIndex iotago.SlotIndex) (iotago.SlotIndex, error) {
-		if startIndex == 0 {
+	sendPreviousStateDiffs := func(startSlot iotago.SlotIndex, endSlot iotago.SlotIndex) (iotago.SlotIndex, error) {
+		if startSlot == 0 {
 			// no need to send previous milestone diffs
 			return 0, nil
 		}
 
 		latestCommitment := deps.Protocol.MainEngineInstance().SyncManager.LatestCommitment()
 
-		if startIndex > latestCommitment.Slot() {
+		if startSlot > latestCommitment.Slot() {
 			// no need to send previous milestone diffs
 			return 0, nil
 		}
 
 		// Stream all available milestone diffs first
 		pruningIndex := deps.Protocol.MainEngineInstance().SyncManager.LatestFinalizedSlot()
```

Comment:
two things, why is it called `pruningIndex` instead of slot and why is it the latest finalized slot? :thinking: